docs/generated/devkit/getProjects.md

```diff
@@ -15,3 +15,7 @@ Use [readProjectConfiguration](../../devkit/documents/readProjectConfiguration)
 #### Returns
 
 `Map`<`string`, [`ProjectConfiguration`](../../devkit/documents/ProjectConfiguration)\>
+
+**`Deprecated`**
+
+This method will be removed in Nx v18. It's usefulness is a bit hampered from including non-project.json projects which are not updateable. Use [getProjectJsonProjects](../../devkit/documents/getProjectJsonProjects) instead.
```

Comment:
I think this function is still useful for code that wants to read all projects, even if the config is not editable.

Reply:
The problem is that this function isn't accurate in terms of "show me all projects" either - inferred projects from other plugins are absent. Package json projects are inferred too, but we have some custom logic to show them thats a bit weird